lib/site_template/_config.yml

```diff
@@ -29,6 +29,15 @@ markdown: kramdown
 theme: minima
 gems:
   - jekyll-feed
-exclude:
-  - Gemfile
-  - Gemfile.lock
+
+# Exclude from processing.
+# The following items will not be processed, by default. Create a custom list
+# to override the default setting.
+# exclude:
```

Comment:
.. I never added that patch to `jekyll-data` plugin because then one would never be able to take an item off the list if they needed to..